roles/docker/tasks/main.yml

```diff
@@ -0,0 +1,133 @@
+---
+- name: install dependencies
+  apt:
+    pkg: "{{ item }}"
+    state: present
+    update_cache: yes
+  with_items:
+    - ca-certificates
+    - curl
+    - python-pip
+    - unattended-upgrades
+- name: add apt key for docker
```

Comment:
No space between sections makes the roles contents hard to read. Good practice would be to have space before every ' -name'

Reply:
Added space between all sections.